mkosi/config.py

```diff
@@ -2879,9 +2890,13 @@ def load_environment(args: argparse.Namespace) -> dict[str, str]:
     if dnf := os.getenv("MKOSI_DNF"):
         env["MKOSI_DNF"] = dnf
 
-    for s in args.environment:
+    environment_values : list[str] = sum([f.read_text().strip().splitlines() for f in args.environment_files], [])
```

Comment:
Why `sum()`? That shouldn't be necessary. Also please just name this `env` instead of `environment_values`. We prefer shorter variable names for local variables.

Reply:
There should also be no need to specify the type explicitly here, so please drop that.